src/java/org/apache/cassandra/config/DatabaseDescriptor.java

```diff
@@ -1899,6 +1899,11 @@ public static IFailureDetector newFailureDetector()
         return newFailureDetector.get();
     }
 
+    public static void setFailureDetector(Supplier<IFailureDetector> fn)
```

Comment:
its not for repair but for other stuff like `StorageService`.  To make sure we don't freak out in random cases, setting to a fake FD lets me make sure nothing happens randomly in the background.

Reply:
I can remove... I remember that I added this as something was failing as being down, but since I didn't document in this code I don't remember if its still an issue or not... I should listen to the advice I am actively giving @aratno ... :sigh: